tests/unit/test_utils.py

```diff
@@ -363,6 +363,37 @@ def test_repack_model_from_s3_saved_model_to_s3(tmpdir):
     assert re.match(r'^s3://fake/model-\d+-\d+.tar.gz$', new_model_uri)
 
 
+def test_repack_model_with_inference_file_named_differently_should_create_inference_file(tmpdir):
+
+    tmp = str(tmpdir)
+
+    model_path = os.path.join(tmp, 'model')
+    write_file(model_path, 'model data')
+
+    source_dir = os.path.join(tmp, 'source-dir')
+    os.mkdir(source_dir)
+    script_path = os.path.join(source_dir, 'another-inference.py')
```

Comment:
thoughts on making these constants?

Reply:
Do you mean:
```python
SCRIPT_PATH = os.path.join(source_dir, 'another-inference.py')
```
inside the function? These vairable is not used anywhere else, there is no need to define it outside the test function.

Reply:
you are referencing another-inference.py pretty often.